pkg/local_object_storage/engine/head_test.go

```diff
@@ -22,20 +22,25 @@ func TestHeadRaw(t *testing.T) {
 
 	parentAddr := addressSDK.NewAddress()
 	parentAddr.SetContainerID(cid)
-	parentAddr.SetObjectID(parent.ID())
+
+	idParent, _ := parent.ID()
+	parentAddr.SetObjectID(idParent)
 
 	child := generateObjectWithCID(t, cid)
 	child.SetParent(parent)
-	child.SetParentID(parent.ID())
+	child.SetParentID(idParent)
 	child.SetSplitID(splitID)
 
 	link := generateObjectWithCID(t, cid)
 	link.SetParent(parent)
-	link.SetParentID(parent.ID())
-	link.SetChildren(*child.ID())
+	link.SetParentID(idParent)
+
+	idChild, _ := child.ID()
+	link.SetChildren(idChild)
 	link.SetSplitID(splitID)
 
 	t.Run("virtual object split in different shards", func(t *testing.T) {
+		t.Skip("not working, see neofs-sdk-go#242")
```

Comment:
Do we have an issue in this repo?

Reply:
No, we'll be motivated to remove test skipping, and this problem is going to be fixed by https://github.com/nspcc-dev/neofs-sdk-go/issues/242